### PR TITLE
Feature/student profile

### PR DESCRIPTION
**Pull request by joaoprferreira**
## Description

eu fiz componente de Perfil do aluno

Provide a description of what was changed and why it was implemented this way.

### Steps

1. List all steps in a numbered list

## Screenshots / Videos

Add any relevant screenshots or videos. Any video should be added to the Jira ticket and linked here.

image

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

File: src/components/StudentProfile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { descriptionProfileMock } from '../../mocks/DescriptionProfile.mock';
+import * as S from './styles';
+
+export const StudentProfile = () => {
+  return (
+    <S.Container>
+      <S.TextBox>
+        <div>
```
Comment (ramonxm) [inline src/components/StudentProfile/index.tsx:8]: Div se faz necessário?

File: src/components/StudentProfile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          <S.TitleProfile>Qual o perfil de aluno que</S.TitleProfile>
+          <S.SubtitleProfile>buscamos?</S.SubtitleProfile>
+        </div>
+        {descriptionProfileMock.map(({ description }, index) => (
+          <S.DescriptionProfile
+            dangerouslySetInnerHTML={{ __html: description }}
+            key={index}
```
Comment (ramonxm) [inline src/components/StudentProfile/index.tsx:15]: Por enquanto podemos deixar assim, mas podemos pensar em uma alternativa melhor ao index

File: src/components/StudentProfile/styles.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  @media (max-width: ${({ theme: { media } }) => media.tablet}) {
+    background: none;
+    flex-direction: column;
+    padding-left: 0px;
+    margin-top: 7.5rem;
+  }
+`;
+
+export const TextBox = styled.div`
+  width: 585px;
```
Comment (ramonxm) [inline src/components/StudentProfile/styles.ts:26]: Use medidas relativas

File: src/mocks/DescriptionProfile.mock.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+type DescriptionProfile = {
+  description: string;
+};
+
+export const descriptionProfileMock: DescriptionProfile[] = [
+  {
```
Comment (ramonxm) [inline src/mocks/DescriptionProfile.mock.ts:6]: Pode ter uma propriedade `key` e se utiliza no lugar do index no map

**Review by ramonxm — CHANGES_REQUESTED** [review]

File: src/components/StudentProfile/styles.ts [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  height: 532px;
```
Comment (jonataspinto) [inline src/components/StudentProfile/styles.ts:27]: Faz sentido limitar a altura?

File: src/components/StudentProfile/styles.ts [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  margin: 20px 0;
+  color: ${({ theme }) => theme.palette.design.white};
+
+  @media (max-width: ${({ theme: { media } }) => media.tablet}) {
+    text-align: center;
+  }
+`;
+
+export const SubtitleProfile = styled(Typography).attrs({
+  variant: 'subtitle1',
```
Comment (jonataspinto) [inline src/components/StudentProfile/styles.ts:51]: As variantes que existiam não atendem?

File: src/components/StudentProfile/styles.ts [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  @media (max-width: ${({ theme: { media } }) => media.tablet}) {
+    width: 25rem;
+  }
+`;
+
+export const TitleProfile = styled(Typography).attrs({
+  variant: 'h2',
+  weight: '600',
+})`
+  margin: 20px 0;
```
Comment (jonataspinto) [inline src/components/StudentProfile/styles.ts:42]: Para 24px são 2rem.

File: src/components/StudentProfile/styles.ts [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+  @media (max-width: ${({ theme: { media } }) => media.tablet}) {
+    text-align: center;
+  }
+`;
+
+export const SubtitleProfile = styled(Typography).attrs({
+  variant: 'subtitle1',
+})`
+  margin-bottom: 80px;
```
Comment (jonataspinto) [inline src/components/StudentProfile/styles.ts:53]: Rem

**Review by jonataspinto — CHANGES_REQUESTED** [review]

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.63 | ✅ |
| First Contentful Paint | 3.4 s |  |
| Time to Interactive | 5.1 s |  |
| Speed Index | 3.9 s |  |
| Total Blocking Time | 370 ms |  |
| Largest Contentful Paint | 5.5 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.95 | ✅ |
| First Contentful Paint | 1.7 s |  |
| Time to Interactive | 5.4 s |  |
| Speed Index | 2.8 s |  |
| Total Blocking Time | 130 ms |  |
| Largest Contentful Paint | 1.8 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/components/StudentProfile/index.tsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
+import { studentProfileSection } from '../../mocks/DescriptionProfile.mock';
```
Comment (jonataspinto) [inline src/components/StudentProfile/index.tsx:1]: Não é mais usado aqui né?

**Review by jonataspinto — APPROVED** [review]
Boa! Só ficou uma coisa para corrigir mas vou deixar o approve já

**Comment by joaoprferreira** [comment]
Estava dando uma investigada relacionado ao score, e realizei uns testes.

- score atual 
- 
atual

Retirei algumas imagens e fui acompanhando o score, a que mais elevou o score foi quando removi o background do header

- score sem imagem do header 

sem a image do header

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.53 | ✅ |
| First Contentful Paint | 3.8 s |  |
| Time to Interactive | 6.2 s |  |
| Speed Index | 4.5 s |  |
| Total Blocking Time | 590 ms |  |
| Largest Contentful Paint | 5.1 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.95 | ✅ |
| First Contentful Paint | 1.6 s |  |
| Time to Interactive | 5.3 s |  |
| Speed Index | 2.5 s |  |
| Total Blocking Time | 130 ms |  |
| Largest Contentful Paint | 2.0 s |  |
| Cumulative Layout Shift | 0.042 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Pull request merged**